simplq/src/components/pages/Admin/TourSteps.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+export const toursteps = [
+    {
+      selector: '[data-tut="reactour__addMember"]',
+      content:  ()=> (
+        <div>
+          <span style={{padding:"70px" }} >
+            <img src=" /images/sharequeueArrow.png"  height="80" width="120" />
+          </span>
+          <br/>
+          <div style={{transform: "rotate(-5deg)" }}>
+            Add users to the queue manually
+          </div>
+        </div>
+      ),
+      style: {
+        fontFamily: "Pacifico",
+        backgroundColor: "transparent",
+        color: "white",
+        fontSize:"25px",
+        boxShadow: "none"
+      },
+      position: 'left'
+    },
+    {
+      selector: '[data-tut="reactour__shareQueue"]',
+      content: () => (
+        <div>
+            <img  src=" /images/sharequeueArrow.png"  height="80" width="120" />
+          <div style={{transform: "rotate(-10deg)"}}>Share Queue</div>
+        </div>),
+        style: {
+                  fontFamily: "Pacifico",
+                  backgroundColor: "transparent",
+                  color: "white",
+                  fontSize:"25px",
+                  boxShadow: "none"
+                },
+                position: 'top'
+    }
+  ];
+
+  export function checkUserTourStatus() {
+    var isuserBeenOnTour = localStorage.getItem("__user_been_on_tour__");
+    if(isuserBeenOnTour === null)
```

Comment:
Can you replace the variable name with hasUserBeenOnTour? Also, to keep things consistent, can you change all the var keywords to const or let as required?

Reply:
I had actually meant the variable renaming the variable isuserBeenOnTour in the function. The function name change is good, Could you change the variable name to tourStatus or something similar? And you can declare with const.